testing/test_schools_ks.py

```diff
@@ -0,0 +1,121 @@
+import csv
+import os
+import subprocess
+import sys
+import xml.etree.ElementTree as ET
+from pathlib import Path
+from pprint import pprint
+from sys import stderr
+
+from utility import position_on_edge
+
+if 'SUMO_HOME' in os.environ:
+    tools = os.path.join(os.environ['SUMO_HOME'], 'tools')
+    sys.path.append(tools)
+else:
+    sys.exit("Please declare environment variable 'SUMO_HOME' to use sumolib")
+
+import sumolib
+
+
+class TestInstance:
+    def __init__(self, name: str, net_file: str, gen_stats_in_file: str, real_stats_file: str, centre):
+        self.name = name
+        self.centre = centre
+        self.net_file = net_file
+        self.gen_stats_in_file = gen_stats_in_file
+        self.real_stats_file = real_stats_file
```

Comment:
```suggestion
class TestInstance:
    def __init__(self, name: str, net_file: str, gen_stats_in_file: str, real_stats_file: str, centre: Tuple[float, float]):
        self.name = name
        self.net_file = net_file
        self.gen_stats_in_file = gen_stats_in_file
        self.real_stats_file = real_stats_file
        self.centre = centre
```
Typehinting and ordering. Might need to import `Typing`

Reply:
This isn't very important but I noticed it.

Reply:
ede971ec3f0a67c1ccdb00163a4d52ec62143464